presets/4.3/other/gyroflow_min_blackbox.txt

```diff
@@ -0,0 +1,22 @@
+#$ TITLE: Gyroflow minimum settings
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: OTHER
+#$ STATUS: COMMUNITY

```

Comment:
should be experimental until more community testing has been vitrified

Reply:
ok, done!